clang/lib/CIR/CodeGen/CIRGenBuiltinX86.cpp

```diff
@@ -1182,6 +1179,25 @@ mlir::Value CIRGenFunction::emitX86BuiltinExpr(unsigned builtinID,
                  std::string("unimplemented X86 builtin call: ") +
                      getContext().BuiltinInfo.getName(builtinID));
     return {};
+  case X86::BI__builtin_ia32_alignd128:
```

Comment:
Please do not change the order of the builtins in the switch. We've intentionally kept the same ordering as classic codegen to make it easier to navigate the code. Instead, duplicate the `errorNYI` handling above the builtins you are handling to prevent a fallthrough.